app/assets/javascripts/views/SourceWindowView.js

```diff
@@ -86,6 +88,11 @@ define([
     hide: function(e) {
       e && e.preventDefault();
       this.model.set('hidden', true);
+
+      //Give back scroll beyond modal window.
+      this.$body.removeClass('is-no-scroll');
+      this.$html.removeClass('is-no-scroll');
```

Comment:
Merging, but will improve this eventually, we can achieve that with a one liner :)